lib/ansible/cli/doc.py

```diff
@@ -1083,7 +1083,7 @@ def format_plugin_doc(plugin, plugin_type, doc, plainexamples, returndocs, metad
             text = DocCLI.get_man_text(doc, collection_name, plugin_type)
         except Exception as e:
             display.vvv(traceback.format_exc())
-            raise AnsibleError("Unable to retrieve documentation from '%s' due to: %s" % (plugin, to_native(e)), orig_exc=e)
+            raise AnsibleError("Unable to retrieve documentation from '%s'" % (plugin), orig_exc=e)
```

Comment:
Would you mind using the `except .. from ..` syntax? It'd be useful for what I'm trying to achieve in #82730.

```suggestion
            raise AnsibleError("Unable to retrieve documentation from '%s'" % (plugin), orig_exc=e) from e
```